src/components/Tabs/Tabs.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Base from '../Base.jsx';
+import style from './style.scss';
+import cn from 'classnames';
+
+class Tabs extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      selectedIndex: 0
+    };
+  }
+  onClick(index) {
+    this.setState({
+      selectedIndex: index
+    });
+    if (this.props.onClick) {
+      this.props.onClick(index);
```

Comment:
i'm wondering if we can somehow give a `key` to each tab, so that what's passed to the handler is like, 'my_tab' and 'another_tab' instead of `0` or `1`. that would be less fragile if we were to add new tabs or reorder existing ones.